src/SFA.DAS.LevyAggregationProvider.Worker.UnitTests/Providers/LevyAggregationTests/WhenIAggregateLevyData.cs

```diff
@@ -223,8 +224,8 @@ public void ThenTheTotalIncludesTheTopUpValueAndTheEnglishPercentage()
             var actual = _levyAggregator.BuildAggregate(actualData);
 
             //Assert
-            var expectedTotal = (actualData.Data[0].LevyDueYtd * actualData.Data[0].EnglishFraction) + (actualData.Data[0].TopUp * actualData.Data[0].EnglishFraction);
-            Assert.AreEqual(expectedTotal, actual.Data[0].Balance);
+            var expectedTotal = Math.Round(actualData.Data[0].LevyDueYtd * actualData.Data[0].EnglishFraction,2) + Math.Round(actualData.Data[0].TopUp * actualData.Data[0].EnglishFraction,2);
```

Comment:
the round early is due to the previous amount already being rounded. This is the correct place for it, however - we are going to delete all this later 👍